weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/traceServerClient.ts

```diff
@@ -94,7 +109,10 @@ export class TraceServerClient extends DirectTraceServerClient {
 
   public feedbackCreate(req: FeedbackCreateReq): Promise<FeedbackCreateRes> {
     const res = super.feedbackCreate(req).then(createRes => {
-      const listeners = this.onFeedbackListeners[req.weave_ref] ?? [];
+      const feedbackTypeResolved =
+        req.feedback_type ?? this.FEEDBACK_TYPE_DEFAULT;
+      const listeners =
```

Comment:
for here and in replace, i think what you want to hit all the listeners for weave_ref + feedback_type AND weave_ref + ''.